src/particle/pdgid/functions.py

```diff
@@ -52,12 +52,14 @@ class Location(IntEnum):
 def is_valid(pdgid):
     # type: (PDGID_TYPE) -> bool
     """Is it a valid PDG ID?"""
-    if _fundamental_id(pdgid) > 0:
+    if _fundamental_id(pdgid) > 0:  # function always returns a positive number
```

Comment:
This comment is slightly confusing. I take it that it returns 0 or a positive number? If so, would `if _fundamental_id(pdgid) != 0:` or even `if _fundamental_id(pdgid):` be clearer?

Reply:
Correct, its >= 0, in other terms the abs of the PDG ID. I will amend to avoid the confusion.